satpy/etc/readers/mirs.yaml

```diff
@@ -0,0 +1,64 @@
+reader:
+  description: NetCDF Reader for the Microwave Integrated Retrieval System Level 2 swath products
+  name: mirs
+  short_name: MiRS Level 2 NetCDF4
+  long_name: MiRS Level 2 Swath Product Reader (NetCDF4)
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
+  sensors: [amsu, amsu-mhs, atms, ssmis, gmi]
+  data_files:
+     - url: "https://zenodo.org/record/4472664/files/limbcoef_atmsland_noaa20.txt"
+       file: "limbcoef_atmsland_noaa20.txt"
```

Comment:
The parameter is `filename` (or at least it should be). It technically isn't necessary here as it should end up being the same as the filename in the URL.

Reply:
I will pull the "file" lines.